### PR TITLE
My changes

### PR DESCRIPTION
**Pull request by Creeds1996**

### CODE REVIEW

**Review by Creeds1996 — COMMENTED** [review]
Hey, duplicate code.

File: world.txt [hunk 1]
```diff
@@ -1 +1,2 @@
-Hello friend!
+Hello there!!
```
Comment (Creeds1996) [inline world.txt:1]: Hey I prefer this line of code.

File: world.txt [hunk 2]
```diff
@@ -1 +1,2 @@
...
+Hello friend!
```
Comment (Creeds1996) [inline world.txt:2]: Remove this please

Comment (crckfx) [reply world.txt:2]: Which line do you want removed?

Comment (crckfx) [reply world.txt:2]: disregard! ^

File: world.txt (hunk 1 above)
Comment (crckfx) [reply world.txt:1]: ok understood

**Review by crckfx — APPROVED** [review]

**Pull request merged**